.circleci/utils.py

```diff
@@ -0,0 +1,8 @@
+import urllib, json, sys, subprocess
+
+def build_and_push_docker(dir, container, tag):
+  build_command = ['docker', 'build', '-t', '%s:%s' % (container, tag), dir]
+  push_command = ['docker', 'push', '%s:%s' % (container, tag)]
+  build_code = subprocess.call(build_command)
+  push_code = subprocess.call(push_command)
+  return 0 if build_code is 0 and push_code is 0 else 1
```

Comment:
Be careful! `x is y` checks if `x` and `y` refer to the same object in memory, `x == y` is about equality. You can use `is` for equality checks for small integers, but it breaks down for larger ones: https://kate.io/blog/2017/08/22/weird-python-integers/